.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        javascript:
+          mass_threshold: 100
+ratings:
+  paths:
+  - src/js/**
+exclude_paths:
```

Comment:
Curious how `paths` above and `exclude_paths` interact.  If we only specify `src/js/**`, why do we also need to exclude these paths (and why don't we need to exclude other paths in `src`, like `src/test-support` that have javascript files?

Are these only directories that might contain javascript files?